app/src/test/java/ch/epfl/sdp/blindwar/UserTest.kt

```diff
@@ -13,6 +13,7 @@ class UserTest {
     private val screenName2 = "Screen Name2"
 
     private var testUser = User(firstName, lastName, email, screenName, null, AppStatistics())
+    private var testUser1 = User(firstName, lastName, email, screenName, null, AppStatistics())
 
```

Comment:
Maybe call the two users "testUser1" and "testUser2". Strange to user number on one user but bot the other.